### ISSUE TITLE
ViT WF42M checkpoints

### ISSUE THREAD

**Issue by Dnekhaev** [deepinsight/insightface#1961]
Will checkpoints for ViT trained on WF42M be available?

**Comment by anxiangsir** [comment]
Sorry, we will not release models trained by WF42M.

**Comment by mnikitin** [comment]
@anxiangsir Hi!

According to your new paper, there are a lot of inter-class conflicts in WF42M (fig. 2a).
In order to tackle such cases you propose to filter out negative logits above some threshold. 
But I didn't find any _-PFC*-_ experiments conducted using original WF42M in the paper.

So my question is what's the reason? You haven't tried it at all?
Or is it because of the fact that in case of WF42M most of inter-class conflicts are caused by single outliers (which don't affect centroids so much) and not by splitted classes?

Thank you!

**Comment by Neltherion** [comment]
@anxiangsir But it seems that the dataset can't even be downloaded. So how can we replicate the VIT models ?
